### ISSUE TITLE
APP_DIRS key is missing from the TEMPLATES setting in the documentation for manual install

### ISSUE THREAD

**Issue by derek-adair** [divio/django-cms#5768]
For me, I was getting this error when following the manual install demo...

```
root@4e82e7bb994f:/code# python ./sandbox/manage.py migrate
    Traceback (most recent call last):
      File "/usr/local/lib/python3.4/site-packages/cms/plugin_pool.py", line 72, in validate_templates
        loader.get_template(template)
      File "/usr/local/lib/python3.4/site-packages/django/template/loader.py", line 46, in get_template
        raise TemplateDoesNotExist(template_name)
    django.template.base.TemplateDoesNotExist: cms/plugins/alias.html
```

Adding **'APP_DIRS':True** to the TEMPLATE settings fixes this for me.

**Comment by czpython** [comment]
I suggest to add a note about `APP_DIRS` as well as the alternative using template loaders which is more common. /cc @evildmp @yakky

**Comment by evildmp** [comment]
This is addressed in #5826, which completely rewrites that section. 

The new installation guide should wherever required refer the reader to Django's documentation rather than attempt to recapitulate it here.
